src/app/file-browser/components/file-viewer/file-viewer.component.ts

```diff
@@ -199,7 +199,7 @@ export class FileViewerComponent implements OnInit, OnDestroy {
 
     let url;
 
-    if (original?.type.includes('pdf')) {
+    if (original?.type.includes('pdf') || original?.type.includes('document')) {
```

Comment:
This doesn't seem like the right approach to me since we already check that `this` is a document at the beginning of `getPdfUrl`.